roles/confluent.zookeeper/handlers/main.yml

```diff
@@ -1,3 +1,5 @@
 ---
 - name: restart zookeeper
   include_tasks: tasks/restart_and_wait.yml
+  when:
```

Comment:
tags don't work in handlers?

Reply:
`skip-tags` don't seem to ... Found [this issue](https://github.com/ansible/ansible/issues/50044) on the topic.